### DISCUSSION TITLE
Atom is not initialized after being unmounted and mounted again

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#1205] [Q&A] [answered]
Here is my atom: `const activeSmaIdAtom = atom(-1);`.
This atom's initial value is then reset by a modal component, the atom is used and then unmounted.
When the atom is mounted again it contains the reset value, not the initial "-1" value?
Is this the expected behavior?  I'd expect the initial value instead,  i.e. the last used atom's value should not be persisted.

**Comment by dai-shi** [comment, marked as answer]
Atom values are in the store, in the Provider component. Unless the Provider component is unmounted, atom values exist forever (in WeakMap though).
